sugar/src/connections/express.ts

```diff
@@ -4,26 +4,25 @@ import GetWebhookRoute from "../routes/PostWebhookRoute.js";
 import * as Sentry from '@sentry/node';
 import {Logger} from "@beemobot/common";
 // ^ This needs to be updated; Probably @beemobot/cafe
-import {TAG} from "../index.js";
+
+import {TAG} from "../constants/logging.js";
 export const server = express()
 
-function init() {
+export function initExpressServer() {
     const username = process.env.BASIC_AUTH_USERNAME
     const password = process.env.BASIC_AUTH_PASSWORD
 
     if (username == null || password == null) {
-        Logger.error(TAG, 'Basic authentication is not configured, discarding request to start.')
+        Logger.error(TAG, 'Basic authentication is not configured, it must be configured for authentication purposes.')
```

Comment:
```suggestion
        Logger.error(TAG, 'Basic authentication is not configured. It must be configured for authentication purposes.')
```
(this is just a nit)